### DISCUSSION TITLE
post/193

### DISCUSSION THREAD

**Discussion by giscus** [kyohoonsim/devsim-blog-front-react#12] [Announcements]
# post/193

여러분 **K-패스**라고 아시나요? 2024년 5월부터 시작된 서비스인데, 대중교통비를 환급해주는 지원사업입니다. 

- K-패스란 무엇인가: <https://test.110.go.kr/data/faqView.do?num=62466&curPage=1&scType=&scText=>

K-패스를 가입한 후 **월 15회 이상** 대중교통을 이용하면 최대 60

https://vault.dev-sim.com/post/193

**Comment by nacl1119** [comment]
진짜 좋네요.대중교통으로 츨퇴근이 된다면 ㅜㅜ

**Reply by kyohoonsim** [reply]
회사와 먼 곳으로 이사하면 됩니다..ㅋㅋ
